astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -125,9 +124,6 @@ def test_swapaxes(self):
         assert q_swapaxes.unit == q.unit
         assert np.all(q_swapaxes.value == q.value.swapaxes(0, 2))
 
-    @pytest.mark.xfail(
-        sys.byteorder == "big" and NUMPY_LT_1_21_1, reason="Numpy GitHub Issue 19153"
```

Comment:
@mhvk , won't think comparison also hold for 1.21.0?

Reply:
Darn, I'm an idiot! I just thought we were dropping support for the oldest numpy version in `numpycompat.py`, but apparently we never had anything particularly bad about numpy 1.20...  Sigh... 

And thanks for catching that. Am happy to push an update with that commit removed...